setup.py

```diff
@@ -19,15 +19,14 @@
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License"
       ],
-      install_requires=['numpy', 'requests', 'pillow', 'tqdm', 'networkx', 'pyopencl', 'PyYAML'],
+      install_requires=["numpy", "requests", "pillow", "tqdm", "networkx", "pyopencl", "PyYAML", "pyobjc-framework-Metal;platform_system=='Darwin'", "pyobjc-framework-Cocoa;platform_system=='Darwin'", "pyobjc-framework-libdispatch;platform_system=='Darwin'"],
```

Comment:
this line is getting a bit long with these, I think its probably fine just to newline split it